lib/scala/tree/src/main/scala/edu/rpi/tw/twxplore/lib/geo/models/domain/Block.scala

```diff
@@ -0,0 +1,10 @@
+package edu.rpi.tw.twxplore.lib.geo.models.domain
+
+import edu.rpi.tw.twks.uri.Uri
```

Comment:
Bunch of unused imports here. Use the "Save Actions" IntelliJ plugin. Once it's installed, go to its settings (search for "Save Actions") and have it to Organize imports and format changed lines on save. It'll clear these out.